website/src/components/common/staggered-scroll-animation.js

```diff
@@ -35,7 +35,7 @@ export default class StaggeredScrollAnimation extends PureComponent {
     duration: PropTypes.number,
     delay: PropTypes.number,
     animateOnce: PropTypes.bool,
-    Container: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+    Container: PropTypes.oneOfType([PropTypes.object, PropTypes.func, PropTypes.string]),
```

Comment:
Perhaps you're looking for `PropTypes.elementType`, going by the way it's being used: 
https://github.com/facebook/prop-types/commit/b67bbd40b5bf8b27475801ecdb7617071ceaccac

Reply:
In this case staggered-scroll-animation will receive a styled-component element which is an object. See screenshot below

<img width="660" alt="Screen Shot 2019-03-26 at 11 14 57 AM" src="https://user-images.githubusercontent.com/2554552/55022754-87a7ab00-4fb8-11e9-9208-6a4c8f9c1123.png">

But I will add PropTypes.elementType to the list of possible options